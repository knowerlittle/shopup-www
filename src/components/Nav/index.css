.redNav {
  background: red;
  display: flex;
  justify-content: center;
  height: 10vh;
}

.nav {
  height: 10vh;
  display: flex;
  justify-content: center;
}

.ul {
	color: white;	
}

.wrapper {
  display: flex;
  padding: 10px 80px;
  width: 100%;
}
.logoContainer,
.menuContainer {
  width: 100%;
}
.logoContainer {
  display: flex;
  justify-content: flex-started;
  align-items: center;
}
.menuContainer,
.menuContainer ul {
  display: flex;
  justify-content: flex-end;
  align-items: center;
}
.menuContainer {
  li{
    margin: 0 15px;
    cursor: default;
  }
}
.current {
  font-weight: bold;
  margin: -1.5px;
}

.accountText {
  color: #9B9B9B;
  font-weight: 500;
}

.signinText {
  font-weight: bold;
  color: #292929;
}

.signinText:hover {
  text-shadow: 0.5px 0.5px 0.5px rgba(34, 34, 34, 0.3)
}

.menuLink:hover {
  text-shadow: 1px 1px 1px rgba(34, 34, 34, 0.6);
}