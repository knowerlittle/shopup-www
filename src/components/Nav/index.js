import React from 'react';
import { Menu, Button, Container } from 'semantic-ui-react';
import { Link } from 'react-router-dom';

const Nav = (props) => {
return(
<Container>
	<Menu secondary size='large' inverted={props.inverted}>
		<Menu.Item as={Link} to='/'>
			<Button primary size='huge' circular>
				Post a Popin
			</Button>
		</Menu.Item>
		<Menu.Menu position='right'>
			<Menu.Item  as={Link} to='/signup'>Sign Up</Menu.Item>
			<Menu.Item  as={Link} to='/login' style={{ marginLeft: '0.5em' }}>Log in</Menu.Item>
		</Menu.Menu>
	</Menu>
</Container>
)
};

export default Nav;