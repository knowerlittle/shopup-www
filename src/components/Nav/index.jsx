import React from 'react';
import PropTypes from 'prop-types';
import Home from 'components/Nav/types/Home';
import SignUp from 'components/Nav/types/SignUp';

const options = {
  home: <Home />,
  signup: <SignUp />,
};

const Nav = props => options[props.nav];

Nav.propTypes = {
  nav: PropTypes.string.isRequired,
};

Nav.defaultProps = {
  nav: 'home',
};

export default Nav;
