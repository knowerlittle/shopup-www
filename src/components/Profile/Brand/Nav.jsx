import React from 'react';
import PropTypes from 'prop-types';
import { NavLink } from 'react-router-dom';
import SVG from 'react-inlinesvg';
import AccountIcon from 'assets/icons/accountCircle.svg';
import CalenderIcon from 'assets/icons/miniCalender.svg';
import MessageIcon from 'assets/icons/ic_comment.svg';
import FindSpaceIcon from 'assets/icons/ic_search.svg';
import styles from 'components/Profile/index.css';

const Nav = ({ brandName }) => (
  <div className={styles.menuOptions}>
    <ul>
      <NavLink to="/brand/profile" className={styles.grey} activeClassName={styles.red} >
        <li>
          <SVG src={AccountIcon} />
          <span>{ brandName }</span>
        </li>
      </NavLink>
      <NavLink to="/brand/bookings" className={styles.grey} activeClassName={styles.red} >
        <li>
          <SVG src={CalenderIcon} />
          <span>Bookings</span>
        </li>
      </NavLink>
      <NavLink to="/brand/messages" className={styles.grey} activeClassName={styles.red} >
        <li>
          <SVG src={MessageIcon} />
          <span>Messages</span>
        </li>
      </NavLink>
      <NavLink to="/brand/spaces" className={styles.grey} activeClassName={styles.red} >
        <li>
          <SVG src={FindSpaceIcon} />
          <span>Find a Space</span>
        </li>
      </NavLink>
    </ul>
  </div>
);

Nav.propTypes = {
  brandName: PropTypes.string.isRequired,
};

export default Nav;
