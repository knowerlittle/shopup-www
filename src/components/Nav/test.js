import React from 'react';
import Nav from './';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

test('<Nav /> renders correctly', () => {
  const tree = shallow(
    <Nav />
  );
  expect(toJson(tree)).toMatchSnapshot();
});