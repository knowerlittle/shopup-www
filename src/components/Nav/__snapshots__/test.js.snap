// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Nav /> renders correctly 1`] = `
<Container>
  <Menu
    secondary={true}
    size="large"
  >
    <MenuItem
      as={[Function]}
      to="/"
    >
      <Button
        as="button"
        circular={true}
        primary={true}
        size="huge"
      >
        Post a Popin
      </Button>
    </MenuItem>
    <MenuMenu
      position="right"
    >
      <MenuItem
        as={[Function]}
        to="/signup"
      >
        Sign Up
      </MenuItem>
      <MenuItem
        as={[Function]}
        style={
          Object {
            "marginLeft": "0.5em",
          }
        }
        to="/login"
      >
        Log in
      </MenuItem>
    </MenuMenu>
  </Menu>
</Container>
`;
